src/env/dev/terraform.tfvars

```
@@ -232,3 +238,11 @@ buyerbanks_function_autoscale_minimum = 1
 buyerbanks_function_autoscale_maximum = 3
 buyerbanks_function_autoscale_default = 1
 buyerbanks_delete_retention_days      = 30
+
+# pagopa-proxy app service
+pagopa_proxy_enabled        = true
+pagopa_proxy_redis_capacity = 0
+pagopa_proxy_redis_sku_name = "Basic"
+pagopa_proxy_redis_family   = "C"
+pagopa_proxy_tier           = "Basic"
+pagopa_proxy_size           = "B1"
```

```suggestion
pagopa_proxy_tier           = "Standard"
pagopa_proxy_size           = "S1"
```
not blocking but...
basic sku does not support vnet integration required to connect to internal apim vnet